### PR TITLE
Less massive headers in pull request template

### PR DESCRIPTION
**Pull request by krichter722**
# What this PR does / why we need it
You don't. It looks nicer in my opinion.

# Which issue this PR fixes
./.

# Special notes for your reviewer
./.

# Checklist
- [x] [DCO](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] CHANGELOG.md was updated

### CODE REVIEW

**Review by torstenwalter — COMMENTED** [review]
No need to update changelog or chart version as you just modify a file outside the chart directory

File: charts/jenkins/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: jenkins
 home: https://jenkins.io/
-version: 3.5.2
+version: 3.5.3
```
Comment (torstenwalter) [inline charts/jenkins/Chart.yaml:4]: This is also not needed. You are not modifying the helm chart ;-)

**Comment by torstenwalter** [comment]
Linter wants to have a top level heading

**Comment by krichter722** [comment]
@torstenwalter Maybe it's not the right change then. And this cosmetic tweak is really turning into work. If this ought to be changed, the linter rules should be adjusted imo and that seems overkill. Feel free to close this if you have the same opinion on this.

Still... I think the PR headers are really jumping into ones face and the use of h1 is not convenient.

**Comment by torstenwalter** [comment]
You could try `<!-- markdownlint-disable MD041 -->`

See https://github.com/DavidAnson/markdownlint#configuration

**Review by torstenwalter — APPROVED** [review]
markdownlint-diable works

**Pull request merged**